packages/bus-http/lib/ActorHttp.ts

```diff
@@ -67,6 +62,40 @@ export abstract class ActorHttp extends Actor<IActionHttp, IActorTest, IActorHtt
     });
     return hash;
   }
+
+  /**
+   * Extract the requested URL from the action input.
+   * @param {RequestInfo | URL} input The request input.
+   * @returns {URL} The extracted URL.
+   */
+  public static getInputUrl(input: RequestInfo | URL): URL {
+    return new URL(input instanceof Request ? input.url : input);
+  }
+
+  /**
+   * Creates an appropriate User-Agent header string for Node.js or browser environment.
+   * Within browser environments, the browser agent header is used as-is to avoid problems.
+   * @returns {string} User agent string
+   */
+  public static createUserAgent(actorName: string, actorVersion: string): string {
+    if (ActorHttp.isBrowser()) {
+      return globalThis.navigator.userAgent;
```

Comment:
If I understand correctly, this will still lead to the user agent to be defined in the fetch input within a browser environment.
I suspect this will lead to CORS errors in browser environments, so we may want to just return `undefined` here, and then not set this in the request input down the line.

Reply:
That is actually a great idea!